### DISCUSSION TITLE
扫描件漏识别

### DISCUSSION THREAD

**Discussion by gongdj** [PaddlePaddle/PaddleOCR#13330] [Q&A]
使用paddle进行扫描件图片识别，存在漏识别的文字，下图中的：“注：鼓励”未识别，有办法解决吗？
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/10403160/214cffed-c95d-438c-8e39-3a9fb79f570d)

使用的paddle版本：
paddleocr              2.8.0
paddlepaddle           2.6.1

**Comment by UserWangZz** [comment]
尝试调整一下输入图像的尺寸，或者设置det_db_score_mode为slow， det_db_unclip_ratio也可以调整。
如果还是不行，就需要微调模型了
